Tests/LinuxMain.stencil

```diff
@@ -0,0 +1,22 @@
+@testable import BSONTests
+@testable import MongoSwiftTests
```

Comment:
I'd think we need to remove this and the next line in order for things to compile, given that those modules don't exist in this library?